src/components/07-form/controls/character-count.njk

```diff
@@ -0,0 +1,35 @@
+<form class="usa-form">
+  <div class="usa-character-count" data-module="usa-character-count">
```

Comment:
Out of curiousity, what is `data-module` doing? Haven't seen this used elsewhere in the design system.

Reply:
Good catch. `data-module` isn't being used at this point. I'll push up an update to remove this attribute from the examples.